### ISSUE TITLE
Sending sms from one user to multiple user.

### ISSUE THREAD

**Issue by mayank12psingh** [Vonage/vonage-java-code-snippets#38]
Want to send SMS by one user to multiple user and then those multiple user reply back for approval to further communication.

**Comment by cr0wst** [comment]
Hi @mayank12psingh 

You may want to take a look at our SMS documentation: https://developer.nexmo.com/messaging/sms/overview

## Sending

In order to send messages to multiple users, you will need to call the `submitMessage` for each user.

See: https://developer.nexmo.com/messaging/sms/building-blocks/send-an-sms

## Receiving

In order to track responses you will need to create a webhook and configure your Nexmo Number to use this webhook when it receives a message.

See: https://developer.nexmo.com/messaging/sms/building-blocks/receiving-an-sms

From there, it's up to you to determine how to act based on what values the received message contains.

I do want to point out these two pages too if you were building some sort of opt-in alert system, especially in the US or Canada:

- LVN (Long Virtual Number) Restrictions: https://help.nexmo.com/hc/en-us/articles/204017023
- Short Code Handling: https://developer.nexmo.com/messaging/us-short-codes/overview

The short version is that the US and Canada have restrictions on A2P (Application to Person) communications:
> A2P: A2P (application-to-person) traffic, such as bulk notifications, should be sent to the US via short code (*Note that Sprint and T-Mobile are heavily blocking Short Code traffic) or Toll Free Number. Nexmo has a number of pre-approved use-case short codes that can be used via our US short code API's . (**Note that Nexmo US short codes cannot be used to send SMS to Canada or other countries)  If none of our pre-approved short code use-cases are appropriate we can assist you in the application process for a dedicated short code.

**Comment by mayank12psingh** [comment]
Thank you.
I have gone through those API's and want to know if i have to handle sending sms through front-end only.
working on Angular-5 and have to send sms to multiple if possible only from front end side.

**Comment by ChrisGuzman** [comment]
We strongly recommend not using the Nexmo APIs from the front end because then users can inspect your webpage and extract your API key and secret.  For that reason it's best to make any API calls to Nexmo from your backend.

**Comment by yallen011** [comment]
Recommendations have been provided for this question. Closing
